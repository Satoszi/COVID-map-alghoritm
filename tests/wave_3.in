12 12 2 45
wave_3
++..++..++..
||++||++||++
||||||||||||
||||||||||||
||||||||||||
||||||||||||
||||||||||||
||||||||||||
||||||||||||
||||||||||||
|||++||++|||
|++..++..++.