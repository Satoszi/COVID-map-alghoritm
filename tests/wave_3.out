9 11
2 11
0 11
11 10
6 10
5 10
4 10
10 9
8 9
7 9
1 9
9 8
3 8
2 8
0 8
11 7
6 7
5 7
4 7
10 6
8 6
7 6
1 6
9 5
3 5
2 5
0 5
11 4
6 4
5 4
4 4
10 3
8 3
7 3
1 3
9 2
3 2
2 2
0 2
11 1
6 1
5 1
4 1
8 0
1 0
